tests/code/test_volunteerRegisterEvent.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from app.models.user import User
+from app.controllers.events.volunteerRegisterEvent import volunteerRegister
+from app.models.event import Event
+from peewee import DoesNotExist
+from app.controllers.events.programEligibility import isEligibleForProgram
+
+@pytest.mark.integration
+def test_noUserVolunteerRegister():
+
+    with pytest.raises(DoesNotExist):
+        volunteer = volunteerRegister("asdkl", 1)
+
+    with pytest.raises(DoesNotExist):
+        volunteer = volunteerRegister(132546, 1)
+
+    with pytest.raises(DoesNotExist):
+        volunteer = volunteerRegister(" khatts", 1)
+
+    with pytest.raises(DoesNotExist):
+        volunteer = volunteerRegister("khatts ", 1)
+
+@pytest.mark.integration
+def test_noEventVolunteerRegister():
+
+    with pytest.raises(DoesNotExist):
+        volunteer = volunteerRegister("khatts", 1500)
+
+    with pytest.raises(DoesNotExist):
+        volunteer = volunteerRegister("khatts", "Event")
+
+    with pytest.raises(DoesNotExist):
+        volunteer = volunteerRegister("khatts", -1)
+
+    with pytest.raises(DoesNotExist):
+        volunteer = volunteerRegister("khatts", 0)
+
+
+@pytest.mark.integration
+def test_volunteerRegister():
```

Comment:
You need to test the cases when someone wouldn't be registered.

Reply:
Added more test cases where someone wouldn't be registered.